### ISSUE TITLE
[BUG] CGI: "Use common Y-scale" don't work.

### ISSUE THREAD

**Issue by onlyjob** [moosefs/moosefs#619]
At least for *number of chunk replications*, "Use common Y-scale" does not work, as all chunkservers have uneven Y-scale.

**Comment by onlyjob** [comment]
Also, state of "Use common Y-scale" is not preserved when a different graph is selected.

**Comment by onlyjob** [comment]
Odd: today "Use common Y-scale" seems to work for *number of chunk deletions*. Could it be that graphs re-scaling sometimes fails when common Y-scale option is activated?

**Comment by onlyjob** [comment]
Yes, "common Y-scale" setting should be remembered (and arguably it should even be the default).

I doubt that occasional malfunction could be a browser problem. (I use Firefox-ESR on Debian.) It seems that "common Y-scale" is unstable when master is under heavy load, even when I wait for a long time for graphs to reload...

**Comment by onlyjob** [comment]
There is definitely a reproducible bug in rendering graphs in "common Y-scale" mode. When I refresh the page (i.e. F5) with that mode active, graphs reload without Y-equalisation so I have to switch "Use common Y-scale" off then turn it on again. And simple browser page reload un-equalise graphs despite having the switch in "on" mode.

**Comment by chogata** [comment]
Thank you for all your observations, we will take a closer look at this in our lab.

**Comment by onlyjob** [comment]
Just confirmed the same issue on 4.57.7: refreshing browser page (charts) clears/reverts "common Y-scale", misleadingly leaving "Use common Y-scale" switch in "on" state.
